oauth/mock_auth0_pkce.py

```diff
@@ -101,12 +114,48 @@ def __init__(self, additional_scope=None, token_duration=0):
 
 
 def make_token(token_claims: dict, token_duration: int = 5, additional_scope: list = None) -> str:
-    additional_scope = additional_scope if additional_scope else []
-    expires_at = time.time() + token_duration
-    headers = dict(alg="RS256", kid="fake_kid")
-    token_claims.update(exp=expires_at, scope=additional_scope)
-
-    token = jwt.encode(claims=token_claims, key="mysecret", algorithm="HS256", headers=headers)
+    additional_scope = additional_scope if additional_scope else ["openid profile email"]
+    now = time.time()
+    issued_at = now
+    expires_at = now + token_duration
+    headers = dict(alg="RS256")
+    token_claims.update(
+        exp=expires_at,
+        iat=issued_at,
+        # For some reason, Auth0 requires the client id here for the audience value even though the Auth0Provider takes
+        # distinct prop values 'audience' and 'clientId'. Without this configuration, the Auth0 React package rejects
+        # the JWT.
+        aud="local-client-id",
```

Comment:
this has to correspond to `clientId` prop value from [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/17e490d8bd1b44074efabe1709db469c61c57331/frontend/src/pages/_app.tsx#L66) and notably _not_ the `audience` prop value immediately above it. 🤬🤦‍♂️

Reply:
Oh wow that's crazy! Thanks for adding the comment for this gotcha 😵‍💫